WooCommerce/Classes/ViewRelated/Products/Add Product/AddProductWithAI/AIToneVoice/AIToneVoiceView.swift

```diff
@@ -0,0 +1,103 @@
+import SwiftUI
+
+/// View to select AI tone and voice
+///
+struct AIToneVoiceView: View {
+    @StateObject private var viewModel: AIToneVoiceViewModel
+
+    @Environment(\.presentationMode) private var presentation
+
+    init(viewModel: AIToneVoiceViewModel) {
+        self._viewModel = StateObject(wrappedValue: viewModel)
+    }
+
+    var body: some View {
+        ScrollView {
+            VStack(alignment: .leading, spacing: Layout.blockVerticalSpacing) {
+                titleBlock
+
+                // Subtitle label.
+                Text(Localization.subtitle)
+                    .foregroundColor(Color(.secondaryLabel))
+                    .bodyStyle()
+                    .padding(.horizontal, Layout.subtitleExtraHorizontalPadding)
+
+                // List of AI tones.
+                ForEach(viewModel.tones, id: \.self) { tone in
+                    VStack(alignment: .leading, spacing: 0) {
+                        SelectableItemRow(
+                            title: tone.rawValue,
+                            selected: tone == viewModel.selectedTone,
+                            displayMode: .compact,
+                            alignment: .trailing)
+                        .onTapGesture {
+                            viewModel.onSelectTone(tone)
+                        }
+
+                        Divider()
+                    }
+                }
+
+                Spacer()
+            }
+            .padding(Layout.defaultPadding)
+        }
+    }
+}
+
+private extension AIToneVoiceView {
+    var titleBlock: some View {
+        ZStack {
+            HStack {
+                Button(action: {
+                    presentation.wrappedValue.dismiss()
+                }, label: {
+                    Image(systemName: "chevron.backward")
+                        .headlineLinkStyle()
+                })
+
+                Spacer()
+            }
+            .padding(.horizontal, Layout.backButtonHorizontalPadding)
+
+            VStack(spacing: 0) {
+                HStack {
+                    Spacer()
+
+                    Text(Localization.title)
+                        .fontWeight(.semibold)
+                        .headlineStyle()
+
+                    Spacer()
+                }
+                .padding(.vertical, Layout.titleVerticalPadding)
+
+                Divider()
+            }
+        }
```

Comment:
💡  We can simplify this by putting the button and the title in the same HStack like so:

```suggestion
        VStack(spacing: 0) {
            HStack {
                // Dismiss button
                Button(action: {
                    presentation.wrappedValue.dismiss()
                }, label: {
                    Image(systemName: "chevron.backward")
                        .headlineLinkStyle()
                })

                Spacer()

                // Title
                Text(Localization.title)
                    .fontWeight(.semibold)
                    .headlineStyle()
                    .padding(.vertical, Layout.titleVerticalPadding)

                Spacer()
            }
            .padding(.horizontal, Layout.backButtonHorizontalPadding)

            Divider()
        }
```

Reply:
❓ Will this perfectly align the `title` at the center? I used a ZStack because this setup doesn't align the text in center of the screen. Please let me know if you think using ZStack isn't needed to get that alignment.

Reply:
To meet the milestone, I will enable auto-merge now. I will make changes if needed in a future PR.

Reply:
From my testing, SwiftUI magically keeps the title in the center ✨